examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -76,6 +76,15 @@ void ModelCommand::Shutdown()
     CHIPCommand::Shutdown();
 }
 
+void ModelCommand::ClearICDEntry(const chip::ScopedNodeId & nodeId)
+{
+    CHIP_ERROR deleteEntryError = CHIPCommand::sICDClientStorage.DeleteEntry(nodeId);
```

Comment:
This could technically return error if we try to delete an entry that was already deleted. 
Do we still want to print the error log?